src/sass/ui.scss

```diff
@@ -622,6 +627,15 @@ body {
 .view__found-x-items,
 .view__found-all-items {
   color: $c-white;
+  height: unset;
+  min-height: 100%;
+  padding: 50px 0 100px;
+
+  &__disclaimer {
+    font-size: 80%;
```

Comment:
Any reason why the percentage here? Instead of just fixed pixels?

Reply:
Done.